modules/gis/src/lib/reproject.js

```diff
@@ -0,0 +1,36 @@
+export function mapBinaryCoords(binaryFeatures, fn) {
```

Comment:
By JS analogy, calling this function `map...` seems potentially confusing since this modifies in place, rather than create a new output data structur, like `array.map()`.

`transformBinaryGeometryCoordinates()`?

Reply:
Would it be preferable not to transform in place? To be clear both the binary and GeoJSON implementations transform the input